test/fail_compilation/imports/diag12598a.d

```diff
@@ -1,3 +0,0 @@
-module imports.diag12598a;
-
-struct lines { }
```

Comment:
It's better to move to `compilable` directory, rather than deleting test case for issue 12598.

Of course adding a comment about that in commit log is even better.